ui/jobs/jobs.js

```diff
@@ -1962,51 +1845,51 @@ class Bars {
         demiSummoningBox.parentNode.classList.remove('last');
     });
 
-    this.abilityFuncMap[kAbility.Miasma] = () => {
+    this.abilityFuncMap[PveAction.ACN.Miasma.HexID] = () => {
       miasmaBox.duration = 0;
       miasmaBox.duration = 30;
     };
-    this.abilityFuncMap[kAbility.Miasma3] = () => {
+    this.abilityFuncMap[PveAction.SMN.MiasmaIii.HexID] = () => {
       miasmaBox.duration = 0;
       miasmaBox.duration = 30;
     };
-    this.abilityFuncMap[kAbility.BioSmn] = () => {
+    this.abilityFuncMap[PveAction.ACN.Summon.HexID] = () => {
       bioSmnBox.duration = 0;
       bioSmnBox.duration = 30;
     };
-    this.abilityFuncMap[kAbility.BioSmn2] = () => {
+    this.abilityFuncMap[PveAction.ACN.SummonIi.HexID] = () => {
       bioSmnBox.duration = 0;
       bioSmnBox.duration = 30;
     };
```

Comment:
These two skills are Bio and BioIi, not Summon.

Reply:
Ooops!  Thank you for pointing that out.